stacker/config/translators/kms.py

```diff
@@ -23,9 +23,24 @@ def kms_simple_decrypt(value):
         # In stacker we would reference the encrypted value like:
         conf_key: !kms us-east-1@CiD6bC8t2Y<...encrypted blob...>
 
-        # The above would resolve to
+        You can optionally store the encrypted value in a file, ie:
+
+        kms_value.txt
+        us-east-1@CiD6bC8t2Y<...encrypted blob...>
+
+        and reference it within stacker:
+
+        conf_key: !kms file://kms_value.txt
+
+        # Both of the above would resolve to
         conf_key: PASSWORD
+
     """
+    if value.startswith('file://'):
```

Comment:
i refactored this to a function that could potentially be used by other translators. i'd wait on making it more complex until we need to